storage/src/backend/registry.rs

```diff
@@ -151,6 +165,14 @@ struct RegistryState {
     cached_redirect: HashCache,
 }
 
+#[derive(Debug, Clone)]
+struct MirrorConfigState {
+    // HTTP scheme like: https, http
+    scheme: String,
+    host: String,
+    registry_header: String,
```

Comment:
This should be a HashMap<String, String> ?

Reply:
Yes, fixed.